apps/fluent-tester/src/TestComponents/Button/ButtonVariantTestSection.android.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, FAB } from '@fluentui-react-native/experimental-button';
+import * as React from 'react';
+import { View } from 'react-native';
+import { commonTestStyles, testContentRootViewStyle } from '../Common/styles';
+import { iconProps } from '../Common/iconExamples';
+
+export const ButtonVariantTest: React.FunctionComponent = () => {
+  const [showFABText, setShowFABText] = React.useState(true);
+  const flipFABcontent = React.useCallback(() => setShowFABText(!showFABText), [showFABText]);
+
+  return (
+    <View style={testContentRootViewStyle}>
```

Comment:
This will mean we lose coverage of FAB on iOS. I would prefer to have a ButtonVariantTestSection.mobile.tsx which android and iOS then re-export.

@saadnajmi @lyzhan7 FYI

Reply:
Generally agree with that sentiment. Mobile specific button variants (FAB) are one of the few cases we _do_ want to not render something that otherwise works on desktop, so moving to platform specific file makes sense. And following `.mobile` and re-export is what I personally like most.

Reply:
While all the changes made (FAB working, mapping of appearances for button/FAB) were made for iOS as well, I have _not_ touched the TesterApp or the Readme for iOS, I feel the final validation for the component (with the appropriate token values) as well as the tester app's look would be better handled by whoever picks up this component for iOS. 
The change from .android to .mobile would be better after that exercise.

Reply:
Generally I agree with that, I think in this case I'd try to leave iOS coverage of FAB since we do have it already. But if the iOS folks are ok with that then that's fine with me.

Reply:
FAB is still being rendered as a part of the ButtonSizeTestSection for iOS, so if _something_ breaks FAB for iOS, we are still in a good position to catch it